db/migrate/20220207045007_migrate_otp_code.rb

```diff
@@ -18,9 +18,16 @@ def up
         )
       )
     end
+
+    remove_column :users, :encrypted_otp_secret
+    remove_column :users, :encrypted_otp_secret_salt
+    remove_column :users, :encrypted_otp_secret_iv
```

Comment:
Actually we could put these in the next PR because we need to have another PR to remove the `encryptor` gem because we don't need it after we run migration.